react/react1/week2/react-week2/src/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import Todo from './components/TodoApp';
+import Timer from './components/TodoTimer';
+import reportWebVitals from './reportWebVitals';
+
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <>
+    <h3 className='TodoTitle'>Todo List</h3>
+    <Timer/>
+    <Todo /> 
```

Comment:
it's a more common practice among React developers, to include all the smaller components into one parent component and put that one component here which usually is `App`